tests/config.json

```diff
@@ -34,20 +34,25 @@
                 "type": "service",
                 "account": "md-ingestion"
             },
-            "topic": "backbeat-test-ingestion",
+            "topic": "backbeat-ingestion",
             "zookeeperPath": "/ingestion",
+            "cronRule": "*/5 * * * * *",
             "sources": [
                 {
-                    "name": "ingest-site-1",
+                    "name": "zenkobucket",
+                    "bucket": "bucket1",
+                    "prefix": "location-name",
```

Comment:
minor: prefix, cronRule, zookeeperSuffix, and raftCount can be removed

Reply:
I would prefer to open a separate pr if this ends up being the only request for changes

Reply:
sounds good